lib/user-methods.js

```diff
@@ -478,98 +482,106 @@ const instanceMethods = {
             .then(() => console.log('User is logged out'))
             .catch((e) => print_error(e));
     },
-        openManagementRealm() {
```

Comment:
Below is a whitespace change except for the `config.sync.user && console.warn !== undefined` line. I advise viewing with "hide whitespace changes" enabled.